pkg/alertmanager/alertstore/bucketclient/protobuf_store.go

```diff
@@ -0,0 +1,95 @@
+package bucketclient
```

Comment:
I assume there's a better place to put this file, perhaps somewhere in `pkg/storage` or `pkg/util`, just tell me where and I'll move it.